### DISCUSSION TITLE
Weekly Updates

### DISCUSSION THREAD

**Discussion by mayrinck** [mayrinck/FOSSonMicrosoftStore#21] [Announcements]
Guys, I was thinking on deliver here quick weekly updates about what's going on with the apps. I know I also do that into the readme file but, maybe here it will be easier to see and discuss, what do you guys think?

**Comment by JuliaAgostino** [comment]
> Guys, I was thinking on deliver here quick weekly updates about what's going on with the apps. I know I also do that into the readme file but, maybe here it will be easier to see and discuss, what do you guys think?

Sure! That sounds like a great idea!
